modules/demo/src/main/scala/lucuma/ui/demo/Demo.scala

```diff
@@ -44,22 +44,8 @@ import react.semanticui.elements.label.LabelPointing
 
 import scala.scalajs.js.annotation._
 
-case class AppContext[F[_]]()(implicit val dispatcher: Dispatcher[F], val logger: Logger[F])
-
-object AppContext {
-  implicit val contextReusability: Reusability[AppContext[IO]] = Reusability.never
-
-  implicit def ctx2Dispatcher[F[_]](implicit ctx: AppContext[F]): Dispatcher[F] = ctx.dispatcher
-  implicit def ctx2Logger[F[_]](implicit ctx:     AppContext[F]): Logger[F]     = ctx.logger
-}
-
-object AppCtx extends Ctx[IO, AppContext[IO]]
```

Comment:
Everything is simpler now, since we don't need a `Dispatcher` nor a `Logger`. We will need a new "sync" typeclass when we make the effect generic.